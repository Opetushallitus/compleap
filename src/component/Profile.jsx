import React from 'react'
import t from 'util/translate'

const Profile = () => <div>{t`CompLeap`}</div>

export default Profile
