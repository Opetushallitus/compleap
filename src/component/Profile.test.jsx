import React from 'react'
import { create } from 'react-test-renderer'

import Profile from 'component/Profile'

describe('Profile', () => {
  it('should render correctly', () => {
    const renderedJSON = create(<Profile/>).toJSON()
    expect(renderedJSON).toMatchSnapshot()
  })
})
