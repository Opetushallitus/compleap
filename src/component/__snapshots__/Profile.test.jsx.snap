// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Profile should render correctly 1`] = `
<div>
  CompLeap
</div>
`;
